File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.cpp
class Solution {
public:
    int countWords(string sentence){
        int no_of_spaces = 0;
        for(char c: sentence){
            if(c==' '){
                no_of_spaces++;
            }
        }
        return no_of_spaces+1;
    }   
    
    int mostWordsFound(vector<string>& sentences) {
        int ans =0;
        for (string s: sentences){
            int no_of_words = countWords(s);
            ans = max(ans, no_of_words);
        }
        return ans;
    }
};

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.py
class Solution:
    def countWords(str_sen : str) -> int:
        ans =0
        for i in range(0,len(str_sen)):
            if(str_sen[i] == ' '):
                ans += 1
        return ans+1
    
    def mostWordsFound(self, sentences: List[str]) -> int:
        ans=0
        for i in range(0, len(sentences)):
            count = Solution.countWords(sentences[i])
            ans = max(ans, count)
        return ans
            
            
        